src/js/stores/ConferenceStore.jsx

```diff
@@ -0,0 +1,57 @@
+import { EventEmitter } from 'events';
+import AuthService from 'util/AuthService';
+import dispatcher from 'dispatcher';
+
+const url = 'https://dev.calligre.com';
```

Comment:
This needs to be updated to stop being hardcoded. I think that was happening today, but I don't know how it changed.